lib/utils/segment.js

```diff
@@ -37,7 +25,7 @@ function track(payload) {
   if (TRACKING_IS_DISABLED) {
     return BbPromise.resolve();
   }
-  return request('https://api.segment.io/v1/track', payload);
+  return request('https://tracking.serverlessteam.com/v1/track', payload);
```

Comment:
Quick question: Is this general purpose tracking or segment related? Just wondering because right now it lives in the `segment.js` file (which is ok).

Other than that we still might to check if the `trackingConfig` is provided by the user because then we can e.g. use the debug Segment setup and won't "pollute" our data by sending everything to the production tracking setup.